lib/message.rb

```diff
@@ -0,0 +1,26 @@
+require 'twilio-ruby'
+
+class Message
+  # Note that I'm using test credentials for sid, auth and from_no
+  # As far as I can tell, there's no way of using live SMS without paying
+
+  def initialize
+    @account_sid = ENV['TWILIO_ACCOUNT_SID']
+    @auth_token = ENV['TWILIO_AUTH_TOKEN']
+    @sender = ENV['MY_FROM_NUMBER']
+  end
+
+  def send(customer, time)
+    message = "Thank you! Your order was placed and will be delivered before #{time+(60*60)}"
```

Comment:
Line is too long. [93/80]<br>Surrounding space missing for operator '+'.<br>Surrounding space missing for operator '*'.